File: DayTwo/Python/LevelTwo/task_five.py
number_one = int(input("Enter the first number: "))
number_two = int(input("Enter the second number: "))
number_three = int(input("Enter the third number: "))

largest = number_one

if(number_two > largest):
    largest = number_two
    
elif((number_three > number_two) and (number_three > largest)):
    largest = number_three

print(f"The largest number = {largest}")

File: DayTwo/Java/LevelTwo/TaskFive.java
import java.util.Scanner;

public class TaskFive{
    public static void main(String... args){
     
    Scanner input = new Scanner(System.in);
    
    System.out.print("Enter the first number: ");
    int numberOne = input.nextInt();
        
    System.out.print("Enter the second number: ");
    int numberTwo = input.nextInt();
    
    System.out.print("Enter the third number: ");
    int numberThree = input.nextInt();
    
    int largest = numberOne;
    
    if(numberTwo > largest){
        largest = numberTwo;    
    }
    else if((numberThree > largest) && (numberThree > numberTwo)){
        largest = numberThree;
    }
    
    System.out.print("The largest number = " + largest);   
    }
}
